examples/scale/backend/backend.go

```diff
@@ -77,8 +80,14 @@ func Run() {
 	fe := pb.NewFrontendClient(feConn)
 
 	go doFetch(cfg, be)
-	go doAssign(be)
-	go doDelete(fe)
+
+	if activeScenario.ShouldAssignTicket {
+		go doAssign(be)
```

Comment:
These drain the matches and deleteIds channels.  Without running them the channels will get clogged.